ozaria/site/views/play/level/PlayLevelView.js

```diff
@@ -1447,10 +1411,12 @@ class PlayLevelView extends RootView {
       const showModalFn = () =>
         Backbone.Mediator.publish('level:show-victory', options)
       this.session.recordScores(this.world.scores, this.level)
-      if (this.level.get('replayable')) {
-        return this.session.increaseDifficulty(showModalFn)
-      } else {
-        return showModalFn()
+      if (this.level.get('ozariaType') !== 'capstone') {
+        if (this.level.get('replayable')) {
+          return this.session.increaseDifficulty(showModalFn)
```

Comment:
This is a bit hard to follow. Could a comment be helpful?

Reply:
I am not sure what this code does, but it triggered victory modal for capstone automatically, so I justed wanted to skip this for capstone.